hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -97,24 +108,63 @@ def get_neighbors(array, i, j, n=1):
     return array[i_start:i_stop, j_start:j_stop]
 
 
-def ref_point_with_max_cc(data_cc: np.array, window_size=1):
-    '''
-    shift determine the window size, n=1 9-pixel window, n=2, 25-pixel window, etc.
-    '''
+def find_ref_point_with_largest_cc(data_cc: np.ndarray, indices: (np.array, np.array),
```

Comment:
The name `find_ref_point_with_largest_cc` sounds very similar to `ref_point_with_max_cc`. What is the fundamental difference between those two functions?

Reply:
Eventually we will want some unit tests for `find_ref_point_with_largest_cc`, but we should agree on the purpose of the function first.